README.md

```diff
@@ -1,7 +1,222 @@
+
+### Installation
+
+1. Get a free Google Developer API Key [here](https://console.developers.google.com/)
+2. Clone the repos if you have not already
+* Shelf Share
+```sh
+git clone git@github.com:Shelf-Share-BEE-Mod3/shelf-share-rails.git
+```
+* Book Buddy
+```sh
+git clone https://github.com/Shelf-Share-BEE-Mod3/book_buddy.git
+```
+3. Install Gems
+```ruby
+bundle exec install
+```
+```ruby
+bundle exec figaro install
+```
+4. Add your both your google client id and google client secret key in `application.yml`
+```ruby
+GOOGLE_CLIENT_ID: = 'ENTER IN YOUR API KEY'
+GOOGLE_CLIENT_SECRET: 'ENTER IN YOUR SECRET API KEY'
+```
+
+
+
+<!-- USAGE EXAMPLES -->
+## Usage
+
+Run ```rails s``` in your terminal and navigate to ```http://localhost:3000/``` in your local browser.
+
+Once you sign in with Google, you will be asked to allow Shelf Share access to your Google Books. This will only be triggered the first time you log in.
+
+Your **Dashboard** is a convenient place to view all the books you have borrowed or lent out, as well as new friend and book requests.
+[![dashboard-screenshot][dashboard-screenshot]](https://example.com)
+
+The **Books** tab is where you will go to find the books your friends own.
+[![books-screenshot][books-screenshot]](https://example.com)
+
+To add books that you own to your shelf, navigate to the **Profile** tab, where you will see buttons to add books to your shelf, update your address, and view the books you have already added to your shelf!
+[![profile-screenshot][profile-screenshot]](https://example.com)
+
+_For more examples, please refer to this demonstration [Documentation](Enter Video Link HERE!)_
+
+
+<!-- ROADMAP -->
+## Roadmap
+
+See the [open issues](https://github.com/Shelf-Share-BEE-Mod3/shelf-share-rails/issues) for a list of proposed features and known issues.
+
+
+
+<!-- CONTRIBUTING -->
+## Contributing
+
+Contributions are what make this community such an amazing and fun place to learn, grow, and create! Any contributions you make are **greatly appreciated**.
+
+1. Fork the Project
+2. Create your Feature Branch ```git checkout -b feature/NewGreatFeature```
+3. Commit your Changes ```git commit -m 'Add some NewGreatFeature'```
+4. Push to the Branch ```git push origin feature/NewGreatFeature```
+5. Open a new Pull Request!
+
+
+
+<!-- LICENSE -->
+## License
+
+DISCLAIMER: The ShelfShare platform is a proof-of-concept and not likely to be maintained over time. Thus, any address information submitted will remain in our database indefinitely unless you delete it.  For concerns regarding your privacy, please do not submit any real location information to this website.
+
+
+<!-- CONTACT -->
+## Contact
+
+Phillip S - [![LinkedIn][linkedin-shield]](https://www.linkedin.com/in/phillipstrom/) - [GitHub](https://github.com/Strompy) - [Email](phillip.strom@gmail.com)
```

Comment:
Should we have emails in here? 
Ian also mentioned linking the Turing Portfolio pages